borgia/urls.py

```diff
@@ -110,6 +110,14 @@
     url(r'^(?P<group_name>[\w-]+)/users/(?P<user_pk>\d+)/supply_money/$',
         UserSupplyMoney.as_view(), name='url_user_supplymoney'),
 
+        #       PRE-ALPHA : Ajout par xlsx
+    url(r'^(?P<group_name>[\w-]+)/users/add_by_list_xlsx/$',
+        AddUsersXlsx.as_view(),
+        name='url_add_users_xlsx'),
+    url(r'^(?P<group_name>[\w-]+)/users/add_by_list_xlsx_download/$',
+        AddUsersDownloadXlsx.as_view(),
+        name='url_add_users_download_xlsx'),
+
```

Comment:
I do believe that the best may be :

/add_by_list/xlsx/
/add_by_list/xlsx/download

to open to further file type :)

Reply:
Good thinking, i will change it right away !